openSUSE-Tumbleweed/config.kiwi

```diff
@@ -36,7 +36,7 @@
     <package name="cracklib-dict-small"/>
     <package name="filesystem"/>
     <package name="openSUSE-release"/>
-    <package name="openSUSE-release-mini"/>
+    <package name="openSUSE-release-ftp"/>
     <package name="shadow"/>
```

Comment:
This is something that I haven't ever really understood, why opensuse-release-mini is in OBS for OBS internal use, but not published in repos. Do you know if there is any difference between ftp release and mini release?

Probably another option would be just to include the preferred choice in the prjconf metadata, this way the just include the opensuse-release which is clenear IMHO.

Reply:
There isn't any difference (the flavors will be gone with the new product converter in tumbleweed if everything works as expected), except that -ftp is the only one present in the repos and thus not an annoying `(System)` package.

Yes, the prjconfig could be used as well, but it's already quite a mess. The approach done in oS:F apparently doesn't work, otherwise we wouldn't need any in oS:F:Containers.